### ISSUE TITLE
ALOpsAppTest - Run all tests for all apps (different test suites)

### ISSUE THREAD

**Issue by fvet** [HodorNV/ALOps#242]
**Is your feature request related to a problem? Please describe.**
Following #43 

In case you've a project with 40+ different apps, each having their test app (with a custom Testsuite), what would be the best option / which parameters to use to run all tests at once (ignoring any MS tests) using only one ALOpsAppTest step?

**Comment by waldo1001** [comment]
What we did:
we created a 41st app (TEST-app) that basically installs all apps, and test apps, and runs all of them.  This extra app has a huge pipeline, admittedly, running all tests sequentionally:
![image](https://user-images.githubusercontent.com/12088142/98004746-93905b80-1df0-11eb-843d-d7c2ca61ed5e.png)

basically admitting full flexibility: running or skipping the necessary tests: skipping might be necessary to be able to run a certain test in the context of other apps as well: an app doesn't have to be dependent of another app to make tests of the app fail.

Makes sense?
